scripts/clean.bash

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
I normally just nuke all `node_modules` with `rm -rf **/node_modules` but we have a few we keep around under examples.

I introduced this script to make it easier to reinstall dependencies and clean the project

```bash
$ ./scripts/clean.bash --node_modules --rust --native --misc
$ ./scripts/clean.bash --all
```